README.md

```
@@ -0,0 +1,257 @@
+### Option C — Minikube
+
+```bash
+minikube start --cpus 4 --memory 8192 --kubernetes-version v1.32.0
+minikube addons enable ingress metrics-server dashboard
+```
+
+---
+
+## Directory Tree
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)